themes/classic/_dev/js/cart.js

```diff
@@ -168,6 +168,49 @@ $(document).ready(() => {
     });
   };
 
+  //Hide 'have a code promo' text if displayed when 'close' is
+  let isPresent = $('.promo-code-button').length,
+      nameLength = 0;
+  
+  $('.cancel-promo').hide();
```

Comment:
If this is the default state you should use css to hide it by default

Reply:
Yes, you're right